howdy/debian/postinst

```diff
@@ -118,6 +118,11 @@ log("Upgrading numpy to the latest version")
 # Update numpy
 handleStatus(subprocess.call(["pip3", "install", "--upgrade", "numpy"]))
 
+log("Installing xlib")
+
+# Install xlib
+handleStatus(subprocess.call(["pip3", "install", "--upgrade", "python3-xlib"]))
```

Comment:
This feels like an external dependency for something relatively few users are going to use, maybe we can make it a config option and ask users to install this themselves? That would also remove a point of failure in installation